src/main/java/org/folio/circulation/domain/LoanAction.java

```diff
@@ -20,4 +20,19 @@
   public String getValue() {
     return value;
   }
+
+  public enum ResolveClaimedReturned {
+    CHECKED_IN_RETURNED_BY_PATRON("checkedInReturnedByPatron"),
+    CHECKED_IN_FOUND_BY_LIBRARY("checkedInFoundByLibrary");
```

Comment:
What is the significance of `checkedIn` as a prefix for these values? it doesn't seem to add much information over the rest of the text.

Reply:
This action was mentioned in https://issues.folio.org/browse/UIU-1218:
![image](https://user-images.githubusercontent.com/9094650/75697780-3c147b80-5cb6-11ea-8016-50576fa62f35.png)

I've decided to use similar actions names.

Reply:
Ok. The way actions are modelled is a bit of debt related to how the action history is created, it is a bit messy.

Why did you use that formatting for the actions rather than the text that Emma asked for directly? Is it because the UI would prefer to use a code / constant like value that can then be translated?

Reply:
I just followed existing approach for loan actions. But I agree, that setting the text is clearer than having a constant and then have a mapping on UI. Should we start using text for these loan actions?

Reply:
I think we should ask the UI folks which they prefer. Both styles are used at the moment I believe.

Reply:
Hi @marcjohnson-kint, I've checked it with @SergiyVSergiyenko, both approaches will require mapping, because such values have to be translate for internationalization, it is just about what key to use for a value.

Reply:
Hmm, ok. Out of interest, how do we translate dynamic values like this?

Reply:
If they are equivalent, I suggest we leave this as is.

Reply:
(Actions and loan action history suffer from significant debt due to how they were originally built, this is a bit of a crutch)

Reply:
There are translations files, with key-value pairs: https://github.com/folio-org/ui-users/blob/master/translations/ui-users/en.json#L29-L39